code/controllers/configuration/entries/game_options.dm

```
@@ -161,8 +161,8 @@
 	config_entry_value = "Central Command has ordered the Epsilon security level on the station. Consider your contracts terminated."
 
 /datum/config_entry/number/station_goal_budget
-	default = 1
-	min_val = 0
+	default = 2
+	min_val = 2
```

`default` is an incredibly misleading var name which I mistook to be the default value in the dynamic refactor. Now is a good opportunity to change it to the correct var- `config_entry_value`

Also, `min_val` should remain 0. You don't want to lock people with config access out of stuff unless it's not actually a "possible" value
```suggestion
	config_entry_value = 2
	min_val = 0
```